src/com/Inheritance/Employee.java

```diff
@@ -0,0 +1,7 @@
+package com.Inheritance;
+
+public class Employee {
+    String name = "Sushant Kandel";
+    String Address = "Gaindakot";
```

Comment:
variable name must not be in capital letter.
```suggestion
    String Address = "Gaindakot";
```